packages/EasyErrorHandler/src/Processors/ErrorCodesGroupProcessor.php

```diff
@@ -60,10 +64,13 @@ public function process(): ErrorCodesDto
     /**
      * @param mixed[] $errorCodeNames
      */
-    private function determineCategoryName(array $errorCodeNames): string
+    private function determineCategoryName(array $errorCodeNames, bool $isEnum): string
     {
         $explodedErrorCodeNames = \array_map(
-            static function ($errorCodeName): array {
+            static function ($errorCodeName) use ($isEnum): array {
+                if ($isEnum === true) {
```

Comment:
I believe this is not needed. We use capitalised enum case names, like `case SomeErrorName = 12345;`.

I believe `SomeErrorName` will be more readable.

Reply:
I don't understand your comment. 
This code needed only to be split by groups later. Nothing happened with names here. 
It will be like in the example: `SomeErrorName`